spec/person_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../person'
+
+describe Person do
```

Comment:
- [ Required ] I  think it would be better if you add a test case to test the `add_rental`, and `can_use_services` methods in the `Person` class. According to the project requirements, all the methods in the `Person` class should be tested. Please see the example below as a suggestion on how to achieve this.

## Example code
```ruby
    it "The add_rental method returns a rental and adds its to the book" do
      book = Book.new("Title", "Author")
      person = Person.new(24, "Bob")

      rental = person.add_rental(book, "2021-05-04")

      expect(person.rentals).to eql [rental]
    end

    # *can_use_services? positive scenario
    it "The can_use_services? method returns true if over age" do
      person = Person.new(19, "Sam", false)
      expect(person.can_use_services?).to be true
    end
```